nucleus/cluster/cli/src/main/java/com/sun/enterprise/admin/cli/cluster/ImportSyncBundleCommand.java

```diff
@@ -132,6 +132,7 @@ public class ImportSyncBundleCommand extends LocalInstanceCommand {
     private File backupDir;
 
     private static final String RENDEZVOUS_PROPERTY_NAME = "rendezvousOccurred";
+    private static final Random random = new SecureRandom();
```

Comment:
Is it nessessary to use a SecureRandom for this?

Reply:
"usually static properties are written in capital letters"

https://www.oracle.com/technetwork/java/codeconventions-150003.pdf
I suppose you mean "Constant", e.g. chapter 9. Random is not a constant, although it is static and final.

Reply:
ad SecureRandom -- see my comment up